### DISCUSSION TITLE
blog/sharing-xaringan-slides/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#115] [Comments]
# blog/sharing-xaringan-slides/

How to share your xaringan slides in style, on social media sites and embedded in your web page.

https://www.garrickadenbuie.com/blog/sharing-xaringan-slides/

**Comment by uriahf** [comment]
This is really cool! I wonder, how can I use `webshot2::rmdshot()` in order to create a shared card from any slide besides the first one?

**Reply by gadenbuie** [reply]
It's not currently possible, but that would be a great feature request for [renderthis](https://github.com/jhelvy/renderthis).

**Comment by uriahf** [comment]
Done: https://github.com/jhelvy/renderthis/issues/59
